src/cli.rs

```diff
@@ -337,6 +337,17 @@ Recommended use is with '-o max' and '--fast'.")
                 .long("zopfli")
                 .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new("iterations")
+                .help("Number of Zopfli iterations [default: 15]")
+                .long_help("\
+Set the number of iterations to use for Zopfli compression. The default number of \
+iterations is 15. Using fewer iterations may speed up compression for large files. This \
+option requires '--zopfli' to be set.")
```

Comment:
We should set `default_value` here, then omit mention of default in the help as clap will show it automatically.

Reply:
Ah, good to know. I'd been thrown off by the `--interlace` help message, which (as its comment notes) doesn't have the default set within clap.

Reply:
Yeah fair enough, that interlace one ended causing a problem but there should be no issue here.